autopilot/contractor.go

```diff
@@ -749,13 +739,16 @@ func (c *contractor) runContractRenewals(ctx context.Context, w Worker, budget *
 		renewed, proceed, err := c.renewContract(ctx, w, ci, budget, renterAddress)
 		if err == nil {
 			renewals = append(renewals, renewal{from: ci.contract.ID, to: renewed.ID})
+			nRenewed++
+		} else if ci.usable {
+			toKeep = append(toKeep, ci)
+			nRenewed++
```

Comment:
What if we don't increment `nRenewed` and keep the contract info for later? That way we would keep renewing until `nRenewed` actually hits the limit and also reflects contracts that actually got renewed.

probably doing `remaining := limit` and decrementing that in a `for remaining > 0` will be cleaner than ranging over `toRenew`